### ISSUE TITLE
Accessibility: The Windows magnifier does not work on Windows desktop

### ISSUE THREAD

**Issue by NathanLovato** [GDQuest/learn-gdscript#1213]
A visually impaired user pointed out that they used the Windows magnifier to enlarge the screen and that it did not work in the desktop version of Learn GDScript, but that it worked in the web version.

After searching online, I could find mentions that the Windows magnifier does not work with Godot 3 Windows desktop exports. We'll have to try and see if it does work with Godot 4 (if anyone has the chance to test this with a game made in Godot 4, feedback would be most welcome) and then this could be addressed by #1187.

**Comment by Razoric480** [comment]
I don't know if Windows 11's magnifier works differently from whatever other windows they've been using, but I had no issue getting it to zoom into the app on either the godot 3 version of the app or the godot 4 port. I don't have an older windows to test with however.
